lib/class-wp-rest-menus-controller.php

```diff
@@ -22,6 +22,76 @@ public function __construct( $taxonomy ) {
 		$this->namespace = '__experimental';
 	}
 
+	/**
+	 * Overrides the route registration to support "allow_batch".
+	 *
+	 * @since 11.5.0
+	 */
+	public function register_routes()
+	{
```

Comment:
```suggestion
	/**
	 * Overrides the route registration to support "allow_batch".
	 *
	 * @since 11.5.0
	 *
	 * @see register_rest_route()
	 */
	public function register_routes() {
```